openedx/features/learner_profile/templates/learner_profile/learner-achievements-fragment.html

```diff
@@ -71,12 +71,14 @@ <h4 class="sr-only">
         % elif own_profile:
             <div class="learner-message">
                 <h4 class="message-header">${_("You haven't earned any certificates yet.")}</h4>
-                <p class="message-actions">
-                    <a class="btn btn-brand" href="${marketing_link('COURSES')}">
-                        <span class="icon fa fa-search" aria-hidden="true"></span>
-                        ${_('Explore New Courses')}
-                    </a>
-                </p>
+                % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
```

Comment:
Probably should add a quick unit test to ensure this doesn't get lost.  Note that the test would ensure that it both appears and doesn't appear as appropriate.

Reply:
Test added if you want to rereview.